ote/src/cljs/ote/views/admin/detected_changes.cljs

```diff
@@ -138,12 +175,17 @@
      [:div
       [:h4 "Päivitä palvelulle päivittäiset hash tunnisteet"]
       [form/form
-       {:update!   #(e! (admin-controller/->UpdateHashCalculationValues %))
+       {:update! #(e! (admin-transit-changes/->UpdateHashCalculationValues %))
         :footer-fn (fn [data]
                      [:span
-                      [ui/raised-button {:primary  true
-                                         :on-click #(e! (admin-controller/->ForceHashCalculationForService))
-                                         :label    "Laske"}]])}
+                      [ui/raised-button {:primary true
```

Comment:
Again, the generic ote buttons should be used: `ote.ui.buttons/save`,`cancel`, `delete` :)
Or create a variant of them in that file.

Reply:
Changed to buttons/save